conode/Makefile

```
@@ -1,54 +1,55 @@
 CONTAINER = conode
 IMAGE_NAME = c4dt/$(CONTAINER)
 TAG = dev-$(shell date +%y%m%d)
-WEBAPP_ASSET := ../webapp/src/assets/config.toml
+WEBAPP_ASSET := ../webapp/src/assets/config.local.toml
+BUILD_DIR := build/conodes
+BUILT := $(BUILD_DIR)/built
 
 all: docker
 
-.PHONY: cothority-pull conode newdb docker docker_run docker_wait docker_stop docker_start docker_clean clean
+.PHONY: newdb docker_run docker_wait docker_stop docker_start docker_clean clean
 
-.DEFAULT_GOAL := docker_run
+.DEFAULT_GOAL := $(BUILT)
 
-utils: cothority-pull
-	@echo "Compiling binaries"; \
-	go build -o build/conode; \
-	cd ../cothority && \
-		go build -o ../conode/build/bcadmin ./byzcoin/bcadmin && \
-		go build -o ../conode/build/csadmin ./calypso/csadmin && \
-		go build -o ../conode/build/phapp ./personhood/phapp
+$(BUILT): build/Dockerfile build/run_nodes.sh
+	make newdb
```

it ensures that sub-`make` are correctly run (right number of concurrent jobs for example).
there is some more to replace below.

```suggestion
	$(MAKE) newdb
```

https://www.gnu.org/software/make/manual/make.html#MAKE-Variable